app/src/main/res/values-it/strings.xml

```diff
@@ -1,23 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="launch_dialog_content">Benvenuto su Extras for Android Auto, un tema per cambiare l\'aspetto di Android Auto
-        \n\n Se vuoi ringraziarmi per il lavoro, puoi farlo con una donazione, dal bottone sotto.
+    <string name="previews_text">🖼 Anteprime! 🖼</string>
+    <string name="launch_dialog_title" translatable="false">Ciao!</string>
+    <string name="launch_dialog_content">Benvenuto in Extras for Android Auto, un tema per cambiare l\'aspetto di Android Auto
+        \n\nSe vuoi ringraziarmi per il lavoro, puoi farlo con una donazione, con il pulsante qui sotto.
         \n Anche una piccola somma sarebbe molto apprezzata 😊</string>
-    <string name="do_not_show_again">Non mostrare di nuovo</string>
+    <string name="do_not_show_again">Non visualizzare di nuovo</string>
     <string name="proceed">➡ Continua ➡</string>
-    <string name="previews_text">🖼 Anteprime! 🖼</string>
     <string name="donate">🎁 Dona 🎁</string>
+    <string name="helpButton">🆘 Aiuto 🆘</string>
+    <string name="helpme">Per installare un extra:\n
+        1. Seleziona Android Auto e poi un extra dall'\elenco\n
+        2. Per evitare errori vedi il simbolo di attenzione accanto ad Android Auto\n
+        3. Seleziona il pulsante in basso e \'Costruisci e installa\'\n
+        4. Ripeti il passo 1 a piacimento
+        5. Riavvia, vai alla scheda Gestione, e abilita gli extra</string>
     <string name="helpme_lite">Per installare un extra:
-        \n1. Guarda il messaggio di attenzione (simbolo di attenzione accanto ad Android Auto) per evitare errori
-        \n2. Seleziona Android Auto e poi un extra dalla lista
-        \n3. Premi il bottone in basso e seleziona \'Install Selected\'
-        \n4. Ripetere i passi 2 e 3 a proprio piacimento
-        \n3. Riavvia il dispositivo, ri-apri Lite, recati alla sezione manager, e abilita gli extra</string>
+        1. Seleziona Android Auto\n
```

Comment:
This translation is not ok. Substratum Lite is not translated in other language, hence a tutorial should not translate options.

Reply:
I use the same scheme of English strings.
Then please update also English to reflect always the same meaning.
Then yopu could update it also in Italian.

Reply:
The thing is Substratum Lite is not translated in other languages. So I give instructions with what's actually written on the buttons. No need to translate those if they are untranslated in the real world.

Reply:
As I explained I replied the same struicture/info of master English language.
If you have additional name (sofwtare)in explanation you should have the same names also in English strings.
Substratum Lite sofwtare name seems not include as info in English language.
If you don't have in English you cannot have more info only in Italian.
The English and Italian languae files should contains same type of info (the differences should are only about translations).

Reply:
I'll try to explain it in another way.

Substratum Lite itself has no translations. Whether you have your phone in Italian or French, the button will remain with "Install selected".

If you teach the user to press "Costruisci e installa" you are giving wrong instructions, because there's no such button.

Reply:
Same applies with "scheda gestione", because the tab will continue to be named "Manager"